pom.xml

```diff
@@ -204,7 +204,20 @@
                 <artifactId>org.apache.sling.commons.classloader</artifactId>
                 <version>1.3.2</version>
                 <scope>provided</scope>
-            </dependency>                                 
+            </dependency>
+            <dependency>
```

Comment:
Please don't depend on oak-core, that limits backwards/forward compatibility way too much.

Reply:
I went for 6.3.0, even though 6.1.0 seems to be the minimum version that supports this. since 6.10 is EoL and probably we dont want to support it anyway.